### PR TITLE
Add prometheus related annotations for firebird and phoenix services

### PR DESCRIPTION
**Pull request by DeLaGuardo**
@DG4227 fill free to continue that branch if it looks good

### CODE REVIEW

File: xapix/charts/firebird/values.yaml [hunk 1]
```diff
@@ -12,3 +12,6 @@ resources:
   limits:
     cpu: 4
     memory: "1Gi"
+annotations:
```
Comment (mreinsch) [inline xapix/charts/firebird/values.yaml]: these values won't get picked up when running `helm upgrade --reuse-value` as we usually do. So if we want to get them applied, we need to document it & do it manually.
Alternatively, we could just "hardcode" them in the deployment files.

Comment (mreinsch) [reply xapix/charts/firebird/values.yaml]: And I believe the port should match the pod's port?

Comment (DeLaGuardo) [reply xapix/charts/firebird/values.yaml]: ok

**Comment by mreinsch** [comment]
it actually requests every 5 secs ...

**Comment by DeLaGuardo** [comment]
Sure, I can change it. 20s?

**Comment by DeLaGuardo** [comment]
@mreinsch livenessProbe added

**Comment by DeLaGuardo** [comment]
btw, I just found that scrape_interval can not be enforced by annotations. Prometheus suggesting to change it in prometheus config itself

**Comment by mreinsch** [comment]
ok, then let's not worry about this and just reduce the logging part. @DeLaGuardo, will you remove the interval annotation again?

**Comment by DeLaGuardo** [comment]
done

**Comment by mreinsch** [comment]
liveness checks and metrics scraping seems to work - @DeLaGuardo could you quickly check if all looks good in Prometheus? Last time I check I somehow couldn't see the metrics.

**Comment by DeLaGuardo** [comment]
I can not see it either

Probably because of problems in metrics-server

```
E1029 09:13:19.468427       1 summary.go:97] error while getting metrics summary from Kubelet ip-172-20-60-82.eu-west-1.compute.internal(172.20.60.82:10255): request failed - "500 Internal Server Error", response: "Internal Error: failed to get imageFs stats: failed to get image stats: rpc error: code = Unknown desc = Error response from daemon: layer does not exist"
```

But I'm not sure. Our prometheus is way beyond default configuration( I need a lot of time to understand what is going on

**Comment by mreinsch** [comment]
hm, ok, it's standard install through prometheus operator helm charts from https://github.com/helm/charts/tree/master/stable/prometheus-operator

**Comment by mreinsch** [comment]
we could also scrap the current install once and install once more to make sure that isn't caused by some old stuff from first install. I put install notes here: https://github.com/xapix-io/devops/blob/master/k8s/PROMETHEUS.md

**Pull request merged**